python/helios/platforms.py

```diff
@@ -19,6 +20,14 @@ class PlatformSettings(PlatformSettingsBase):
     y: float = 0
     z: float = 0
 
+    def force_on_ground(self, scene: _helios.Scene):
+        """
```

Comment:
Why does the platform grounding not depend on the strategy enum? I assume that it would use the same algorithms under the hood?